app/controllers/application_controller.rb

```diff
@@ -1,2 +1,15 @@
 class ApplicationController < ActionController::Base
+  helper_method :current_user
+
+  def current_user
+    if session[:user_id]
+      begin
+        @current_user ||= User.find(session[:user_id])
+      rescue ActiveRecord::RecordNotFound
```

Comment:
The if condition is necesarry for find out if the user has a current sesion, that validation within the body was necessary because when the "current_user" is required it can exist in the session context but no always in the DB context (for example if the user was deleted while its session was active).

Reply:
@JsMontesR Good ✅